src/bnb_trading/data_fetcher.py

```
@@ -282,50 +276,52 @@
         """
         Връща най-новата цена на BNB
-        
+
         Returns:
             Последната Close цена
         """
         try:
             ticker = self.exchange.fetch_ticker(self.symbol)
-            return ticker['last']
+            return ticker["last"]
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)